### ISSUE TITLE
When viewing the members list in a group, their names aren't displayed properly

### ISSUE THREAD

**Issue by florianionascu7** [UnigramDev/Unigram#1396]
When opening Unigram and going to a group chat, then clicking on its profile to view the members list, the application doesn't show their names corectly. Instead of a member name, a list item contains the following code:
chatMember {
  user_id = 513305720
inviter_user_id = 0
  joined_chat_date = 1556032327
  status = chatMemberStatusMember {
  }
  bot_info = null

Please fix this issue.

**Comment by MarcAnt01** [comment]
I cannot reproduce

**Comment by FrayxRulez** [comment]
I guess he’s talking about accessibility!

**Comment by MarcAnt01** [comment]
> I guess he’s talking about accessibility!

Yes, probably, just wanted to make sure

**Comment by florianionascu7** [comment]
yeah, I am talking about accessibility. I don't know why information is repported in this way by the screen reader I use. It didn't use to have this behavior on previews Unigram versions such as the version that was available in January.

**Comment by aamir51** [comment]
Hello, this is not just limited to members, happens as well when you try to view the list of files. not of unnecessary information is spoken before the actual file name.

**Comment by sukiletxe** [comment]
It also happens when searching (in the main window) when announcing messages.

**Comment by FrayxRulez** [comment]
Fixed!

**Comment by sukiletxe** [comment]
Still happens with search (with both files and text), I am using latest beta. If it is fixed in code but not pushed yet sorry for bringing it up.
